frontend/js/recordings.js

```diff
@@ -3,7 +3,32 @@ import SimpleTemplate from "./simple-template.js";
 // the specific URL for a given wordform (refactored from previous commits).
 // TODO: should come from config.
 const BASE_URL = "https://speech-db.altlab.app";
-const BULK_API_URL = `${BASE_URL}/api/bulk_search`;
+const LANGUAGE_CODE = getLanguageCodeFromLocation();
+const BULK_API_URL = `${BASE_URL}/${LANGUAGE_CODE}/api/bulk_search`;
+
+
+function getLanguageCodeFromLocation() {
+    const location = window.location.toString();
+    switch (location) {
+        case location.includes("itwewina"):
+            return "maskwacis";
+            break;
+        case location.includes("itwiwina"):
+            return "woodscree";
+            break;
+        case location.includes("gunaha"):
+            return "tsuutina";
+            break;
+        case location.includes("arpeng"):
+            return "arapaho";
+            break;
+        case location.includes("guusaaw"):
+            return "haida";
+            break;
+        default:
+            return "maskwacis"
+    }
```

Comment:
JS case statements are evaluated for whether they're strictly equal to (`===`) to the switch expression. So the case statements are evaluating whether `location === location.includes('itwewina')`, which will always be false (since `location` is a string and `location.includes()` returns a boolean). So this switch block will always hit the `default` statement and return `maskwacis`.

Also, the return statement will automatically break out of the switch statement, so the `break` statements will never be reached and can be removed.

I suggest this approach instead:

```js
if (location.includes(`itwewina`) return `maskwacis`;
// an else statement isn't necessary since you're doing an early return if there's a match
if (location.includes(`itwiwina`) return `woodscree`;
...
```

Reply:
I had a feeling something was off with that switch statement, thanks for catching that!